src/Services/Flight/src/Flight/GenericValueObject.cs

```diff
@@ -0,0 +1,13 @@
+namespace Flight;
+public record GenericValueObject<T>(T Value)
```

Comment:
Thank you for your feedback. I will make the necessary changes as recommended. I appreciate your guidance and will update the code accordingly. I haven't seen the comment on my last commit, otherwise I would have applied it.

Reply:
done